spec/components/restructured_work_history/review_component_spec.rb

```diff
@@ -53,6 +55,16 @@
     )
   end
 
+  context 'when the application is incomplete' do
+    context 'when we click from the application review page' do
+      it 'renders incomplete component with correct path' do
+        result = render_inline(described_class.new(application_form: application_form_with_no_breaks, show_incomplete: true, return_to_application_review: true))
+
+        expect(result.to_s).to include(candidate_interface_restructured_work_history_review_path('return-to' => 'application-review'))
```

Comment:
This feels a bit like copying the implementation in the test — not always a bad idea, but always worth questioning! Do we care about the whole path, or just that we've got the query string on the end? Could we get away with just asserting on the substring `return-to=application-review`.